src/Sylius/Component/Mailer/Repository/EmailRepositoryInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Mailer\Repository;
+
+/**
+ * @author Manuel Gonzalez <mgonyan@gmail.com>
+ */
+interface EmailRepositoryInterface
```

Comment:
Because this PR is still working in progress ; - ), but thank you for your comment